src/components/AccessModeSelection/AccessModeSelection.scss

```diff
@@ -1,33 +1,69 @@
+@import '../../constants/style';
+
 .access-mode-selection {
   display: flex;
   padding-top: 16px;
   padding-bottom: 16px;
   max-width: 420px;
-  box-shadow: 0 4px 8px 0 rgba(39,50,47,.15);
   min-height: 110px;
 }
 
 .access-mode-selection__radio {
   display: flex;
   flex: 1 1 100%;
-  padding-left: 24px;
-  padding-right: 24px;
 }
 
 .access-mode-selection__radio + .access-mode-selection__radio {
-  border-left: 1px solid #eaeaea;
+  margin-left: $default-margin;
 }
 
+#access-mode-selection__private, #access-mode-selection__public {
```

Comment:
This is the AccessModeSelectRadio (the id was already defined). Line 20 is there so that the circle of the radio button is not displayed. In line 38 the design of the selected button is defined. So both are needed.